apps/web/src/pages/TalentNominations/NominationGroup/Profile.tsx

```diff
@@ -1,40 +1,212 @@
+    ...TalentManagementPreferencesOptions
   }
 `);
 
 interface TalentNominationGroupProfileProps {
-  query: FragmentType<typeof TalentNominationGroupProfile_Fragment>;
+  nomineeId: string;
+  communityId: string;
+  shareProfile?: boolean;
+  defaultOpen?: boolean;
 }
 
 const TalentNominationGroupProfile = ({
-  query,
+  nomineeId,
+  shareProfile,
+  defaultOpen = false,
+  communityId,
 }: TalentNominationGroupProfileProps) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const talentNominationGroup = getFragment(
-    TalentNominationGroupProfile_Fragment,
-    query,
+  const intl = useIntl();
+  const [{ data, fetching, error }] = useQuery({
+    query: Nominee_Query,
+    variables: { nomineeId },
+    pause: !shareProfile,
+  });
+
+  const [openSections, setOpenSections] = useState<string[]>(
+    defaultOpen ? Object.values(SECTION_KEY) : [],
   );
+  const hasOpenSections = openSections.length > 0;
+
+  const toggleSections = () => {
+    const newValue = hasOpenSections ? [] : Object.values(SECTION_KEY);
+    setOpenSections(newValue);
+  };
 
-  return null; // TO DO: Add page in #12853
+  return (
+    <Pending fetching={fetching} error={error}>
+      <CardBasic data-h2-border-radius="base(6px 6px 0 0)">
+        <div
+          data-h2-display="base(flex)"
+          data-h2-align-items="l-tablet(flex-end)"
+          data-h2-flex-direction="base(column) l-tablet(row)"
+          data-h2-justify-content="base(space-between)"
+          data-h2-gap="base(x.5 0) l-tablet(0 x.5)"
+          data-h2-margin-bottom="base(x1)"
+        >
+          <Heading
+            Icon={UserCircleIcon}
+            color="primary"
+            data-h2-margin="base(0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Profile and career plan",
+              id: "mSj3fM",
+              description:
+                "Heading for nominee profile page accordion sections",
+            })}
+          </Heading>
+          <div
+            data-h2-display="base(flex)"
+            data-h2-align-items="base(flex-end)"
+            data-h2-gap="base(0 x.5)"
+          >
+            <Button mode="inline" color="secondary" onClick={toggleSections}>
+              {hasOpenSections
+                ? intl.formatMessage({
+                    defaultMessage:
+                      "Collapse all <hidden>profile and career plan</hidden>sections",
+                    id: "/UoSSQ",
+                    description:
+                      "Button text to close all profile and career plan accordions",
+                  })
+                : intl.formatMessage({
+                    defaultMessage:
+                      "Expand all <hidden>profile and career plan</hidden>sections",
+                    id: "tS5VSg",
+                    description:
+                      "Button text to open all profile and career plan accordions",
+                  })}
+            </Button>
```

Comment:
If you don't have access, I don't think we should show this button.

![2025-04-23_15-06](https://github.com/user-attachments/assets/97950439-e0a9-4aaf-bfd6-0c5245d027f9)

Reply:
Fixed in [Refactor open sections state logic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/0b485aba94d5663ac710573631bc2ea1a6839684)

Reply:
Nice, I just noticed the `div` wrapping this button. Is that doing anything? :thinking: I wouldn't expect it to have any affect on display and we might be able to remove it.

Reply:
Oops, [Remove div wrapping button](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/d81d66fe6fa388e10e2d5a59181e892d19842a60)